src/styles/ui-playground/_shared.scss

```diff
@@ -0,0 +1,168 @@
+@use '../reset';
+// TODO: Remove dependencies here
+@use '../sidebar/elements';
+@use '../variables' as var;
+
+@use '../shared';
+
+body {
+  font-size: 100%;
+}
+
+h1 {
+  font-size: 1.75em;
+  font-weight: bold;
+}
+
+h2 {
+  font-size: 1.25em;
+  width: 100%;
+  border-left: 6px solid var.$color-brand;
+  font-weight: bold;
+  padding-left: 0.5em;
+}
+
+h3 {
+  font-size: 1.125em;
+  font-weight: normal;
+  font-style: italic;
+  margin: 1em 0;
+}
+
+h4 {
+  font-size: 1em;
+  font-weight: 500;
+  font-style: italic;
+}
+
+pre {
+  margin: 0;
+}
+
+// Utilities
+.u-center {
+  align-self: center;
+}
+
+// Component styles
+.PlaygroundApp {
```

Comment:
Given the accumulation of styles here I think it would be worth starting to restructure the styles here to match our conventions for other apps - ie. with a `components/` directory and one SASS module per corresponding JS module.